components/Navbar/NavMenuItems.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+
+import Link from 'next/link';
+
+import { menu } from './menu';
+import { usePathname } from 'next/navigation';
+
+const NavMenuItems = () => {
+  const pathname = usePathname();
+  return (
+    <>
+      {menu.map((item) => (
+        <Link
+          key={item.href}
+          href={item.href}
+          className={`py-2 max-sm:px-4 ${
+            pathname === item.href
+              ? 'font-semibold text-primary dark:text-primary-dark max-sm:rounded-3xl max-sm:bg-primary max-sm:text-white-800 max-sm:dark:bg-primary-dark'
```

Comment:
```suggestion
              ? 'customBold text-primary dark:text-primary-dark max-sm:rounded-3xl max-sm:bg-primary max-sm:text-white-800 max-sm:dark:bg-primary-dark'
```

```js
// globals.css
@layer {
  .customBold {
    text-stroke: 1px;
    -webkit-text-stroke: 1px;
  }
  ```
  
  There's a slight jitter and shift when you add bold. This is because bolded text is actually bigger in size. I suggest a small change use a stroke outline instead of bold to show navbar selections so that the layout doesn't have that tiny shift.
  
  Small change- but worth doing especially if you ever add a bold effect on-hover to a navmenu.